packages/mcontrolcenter-bin/mcontrolcenter-bin.pacscript

```diff
@@ -0,0 +1,23 @@
+name="mcontrolcenter-bin"
+gives="mcontrolcenter"
+version="0.3.2"
+depends="libqt5widgets5"
+url="https://github.com/dmitry-s93/MControlCenter/releases/download/${version}/MControlCenter-${version}.tar.gz"
+repology=("project: mcontrolcenter")
+description="An application that allows you to change the settings of MSI laptops."
+hash="bab563f692fb3d42494d95de21611d230e0b0736e2e80f08cf8a1b7b990d9f0f"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
+
+install() {
+	cd app
+	sudo install -Dm755 ${gives} "$pkgdir"/usr/bin/${gives}
+	sudo install -Dm644 ${gives}.desktop $pkgdir/usr/share/applications/${gives}.desktop
+	sudo install -Dm644 ${gives}.svg "$pkgdir"/usr/share/icons/hicolor/scalable/apps/${gives}.svg
+	sudo install -Dm755 ${gives}-helper "$pkgdir"/usr/libexec/${gives}-helper
+	sudo install -Dm644 ${gives}-helper.conf "$pkgdir"/usr/share/dbus-1/system.d/${gives}-helper.conf
+	sudo install -Dm644 ${gives}.helper.service "$pkgdir"/usr/share/dbus-1/system-services/${gives}.helper.service
+}
+postinst() {
+	echo "To start this programme of startup, type:
+	cp /usr/share/applications/mcontrolcenter.desktop ~/.config/autostart/mcontrolcenter.desktop"
```

Comment:
Why did you remove the prompt?

Reply:
Because doing so, if one doesn't have a autostart folder, creating it from the `postinst` hook will make it under root user, also there'll be permission issues later on, as other apps also use that directory. I don't think interfering into someone's home directory, as a root user a good thing. Those apps which need the directory, often make them during the first run.

Reply:
You can use `sudo -u ${PACSTALL_USER} <cmd>` to de-escalate your privileges.

Reply:
You should probably add a prompt to ask if they want to install, *however*, print out the command that will run before.

Reply:
> You can use `sudo -u ${PACSTALL_USER} <cmd>` to de-escalate your privileges.

No help
`````
Do you want this programme to start at startup? [Y/n] y
cp: cannot create regular file '/home/soumyaxubuntu/.config/autostart/mcontrolcenter.desktop': Permission denied

Reply:
The code
````
postinst() {
	if ask "Do you want this programme to start at startup?" Y; then
	sudo -u ${PACSTALL_USER} cp /usr/share/applications/mcontrolcenter.desktop ~/.config/autostart/mcontrolcenter.desktop
	echo "Copied desktop file to ~/.config/autostart folder, programme will now start upon startup."
	else
		echo "Programme will not start upon startup"
	fi
}

Reply:
Ok, so create the directory before copying.

Reply:
I already have it anyways

Reply:
Have what?